Clients/GUI/Pages/InstrumentInfo.razor

```diff
@@ -0,0 +1,469 @@
+﻿@page "/instrumentInfo"
+@using System.Drawing
+@using ChartJs.Blazor.ChartJS.Common
+@using ChartJs.Blazor.ChartJS.Common.Axes
+@using ChartJs.Blazor.ChartJS.Common.Axes.Ticks
+@using ChartJs.Blazor.ChartJS.Common.Enums
+@using ChartJs.Blazor.ChartJS.Common.Properties
+@using ChartJs.Blazor.ChartJS.Common.Time
+@using ChartJs.Blazor.ChartJS.LineChart
+@using ChartJs.Blazor.Charts
+@using ChartJs.Blazor.Util
+@using GUI.Scripts
+@inject NavigationManager NavigationManager
+@inject ISessionStorageService Storage
+@inject IModalService Modal
+
+<div class="modal-body row">
+    <div class="col-md-6">
+        <div style="position: fixed; height:22vh; width:44vw">
+            <ChartJsLineChart @ref="lineChartJs" Config="@lineConfig" Width="300" Height="150" />
+        </div>
+    </div>
+    <div class="col-md-6">
+        <h3 align="center">@instrument.Name</h3>
+        <br>
+        <div class="container">
+            @if (isInstrumentLoading)
+            {
+                <div class="spinner-border" role="status">
+                    <span class="sr-only">Loading...</span>
+                </div>
+            }
+            else if (!errorOccured)
+            {
+                <div>
+                    <div class="form-row input-group-prepend">
+                        <label class="col-3">Current balance </label>
+                        <input type="text" disabled="disabled" class="col-3" @bind-value=@currency>
+                        <input type="number" disabled="disabled" class="col-3" @bind-value=@currentBalance>
+                    </div>
+                    <br>
+                    <div class="form-row input-group-prepend">
+                        <label class="col-3">Update balance</label>
+                        <input type="number" class="col-3" @bind-value=@cash>
+                        <button class="btn btn-primary col-3" value="Update" @onclick="@UpdateBalance">Update</button>
+                    </div>
+                </div>
+                <br>
+
+                <table class="table table-bordered">
+                    <thead class="thead-light">
+                        <tr>
+                            <th scope="col">Figi</th>
+                            <th scope="col">Name</th>
+                            <th scope="col">Type</th>
+                            <th scope="col">Currency</th>
+                            <th scope="col">Count in lot</th>
+                        </tr>
+                    </thead>
+                    <tbody>
+                        <tr>
+                            <td>@instrument.Figi</td>
+                            <td>@instrument.Name</td>
+                            <td>@instrument.Type</td>
+                            <td>@instrument.Currency</td>
+                            <td>@instrument.CountInLot</td>
+                        </tr>
+                    </tbody>
+                </table>
+
+                <div class="form-row">
```

Comment:
It hovers graph, looks not good, but why not

Reply:
What exactly do you not like?